central/tlsconfig/tlsconfig_test.go

```diff
@@ -257,3 +258,30 @@ func writeKey(dir string, privateKey *ecdsa.PrivateKey) error {
 	}
 	return pem.Encode(keyFile, &pem.Block{Type: "EC PRIVATE KEY", Bytes: privateKeyBytes})
 }
+
+func Test_isValidAdditionalCAFileName(t *testing.T) {
+	tests := []struct {
+		name string
+		want bool
+	}{
+		{"bla.crt", true},
+		{"bla.pem", true},
+		{"bla", false},
+		{"bla.crt.", false},
+		{"bla.pem.", false},
+		{"bla.crt.foo", false},
+		{"", false},
+		{"foo.bar", false},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := isValidAdditionalCAFileName(tt.name); got != tt.want {
+				t.Errorf("isValidAdditionalCAFileName() = %v, want %v", got, tt.want)
```

Comment:
Would this work? It safes an if statement by using the assertion instead.

```suggestion
			assert.Equal(t, tt.want, isValidAdditionalCAFileName(tt.name), "isValidAdditionalCAFileName() = %v, want %v", got, tt.want)

```